packages/dcos-integration-test/extra/test_helpers.py

```diff
@@ -75,7 +76,7 @@ def marathon_test_app(
     if network == marathon.Network.USER:
         assert host_port != 0, 'Cannot auto-assign a port on USER network!'
 
-    test_uuid = uuid.uuid4().hex
+    test_uuid = uuid.uuid4().hex if app_prefix is None else "{}-{}".format(app_prefix, uuid.uuid4().hex)
```

Comment:
This change breaks `test_ipv6` test case. Would you please add `app_prefix` to id and keep test_uuid as is? We use test_uuid as an env variable, see `DCOS_TEST_UUID` [below](https://github.com/dcos/dcos/pull/3043/files#diff-fb347b4b1e1d7ab329071b182c56d964L90).